.github/workflows/run-ui-tests.yaml

```diff
@@ -0,0 +1,51 @@
+name: Run UI Tests
```

Comment:
I am not sure that we need it because there are no UI tests in the Fortran plugin.

Reply:
Indeed those are from the template. Just keep it in the history if we need to fetch it later.